library/translation.inc.php

```diff
@@ -321,4 +321,21 @@ function mb_strpad($input, $length, $pad = ' ', $type = STR_PAD_RIGHT, $charset
 return $output;
 }
 
+
+/**
+ * Set theme for RTL language direction in globals setting.
+ * @return {void}
+ * @author Amiel <amielel@matrix.co.il>
+ */
+function setRtlCssTheme(){
+    
+    //validate that user id exist 
+    if(!isset($_SESSION['authUserID']))return;
+     
+    if(!function_exists('setUserSetting'))require_once dirname(__FILE__) . '/user.inc';
+    
+    //set rtl theme 
+    setUserSetting('global:css_header', 'style_sky_blue_rtl.css', null, false); 
+}
```

Comment:
It appears they need to be separate. So, in that case would have the globals selector for the "Theme for RTL Languages" show themes that start with 'style_' and end with '_rtl'. To see how this type of mechanism works for the showing the current LT Themes, see here:
https://github.com/openemr/openemr/blob/master/interface/super/edit_globals.php#L486-L511
(so will basically make a new 'css_ltr' $fldtype to support this)